backend/migrations/versions/63fc48273e15_add_events_table.py

```diff
@@ -0,0 +1,35 @@
+"""Add Events Entity
+
+Revision ID: 63fc48273e15
+Revises: 41ce034b593f
+Create Date: 2023-10-22 21:31:55.595465
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '63fc48273e15'
+down_revision = '41ce034b593f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('event',
+        sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column('name', sa.String(), nullable=False),
+        sa.Column('time', sa.DateTime(), nullable=False),
+        sa.Column('location', sa.String(), nullable=False),
+        sa.Column('description', sa.String(), nullable=False),
```

Comment:
@KrisJordan needs to look into whether this column type is correct. This needs to be a SQL Text field with arbitrary length available.